apps/src/util/formatValidation.js

```diff
@@ -3,7 +3,34 @@
 
 const EMAIL_REGEX =
   /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-export const isEmail = value => EMAIL_REGEX.test(value);
+
+export const isEmail = address => {
```

Comment:
I updated this function to more closely match that on the server side instead of just using this regex, and copied nearly all of the test example emails accordingly.

Reply:
This looks great! One request; could you add a comment both here and in `lib/cdo/email_validator.rb` indicating that these two methods are meant to mirror each other, and asking future devs making changes in either place to maintain that? Thanks!

Reply:
Good call! Thanks!